front-page.php

```diff
@@ -0,0 +1,508 @@
+<?php
+get_header();
+
+// Splash
+$splash_id = IGV_get_option('_igv_site_options', '_igv_front_page_splash_id');
+$splash_link = IGV_get_option('_igv_site_options', '_igv_front_page_splash_url');
+
+// Intro
+$organizers_en = IGV_get_option('_igv_site_options', '_igv_front_organizers_en');
+$organizers_es = IGV_get_option('_igv_site_options', '_igv_front_organizers_es');
+$headline_en = IGV_get_option('_igv_site_options', '_igv_front_headline_en');
+$headline_es = IGV_get_option('_igv_site_options', '_igv_front_headline_es');
+$intro_en = IGV_get_option('_igv_site_options', '_igv_front_description_en');
+$intro_es = IGV_get_option('_igv_site_options', '_igv_front_description_es');
+
+// Visitor Info
+$schedule = IGV_get_option('_igv_visitor_info_options', '_igv_schedule');
+$tickets = IGV_get_option('_igv_visitor_info_options', '_igv_ticket_info');
+$venue_name = IGV_get_option('_igv_site_options', '_igv_venue_name');
+$venue_address = IGV_get_option('_igv_visitor_info_options', '_igv_venue_address');
+
+// Exhibitors
+$publish_exhibitors = IGV_get_option('_igv_exhibitors_options', '_igv_publish_exhibitors');
+
+$exhibitors_text_en = IGV_get_option('_igv_exhibitors_options', '_igv_exhibitors_temp_text_en');
+$exhibitors_text_es = IGV_get_option('_igv_exhibitors_options', '_igv_exhibitors_temp_text_es');
+
+$apply_url = IGV_get_option('_igv_site_options', '_igv_apply_url');
+$apply_end = IGV_get_option('_igv_site_options', '_igv_apply_end');
+
+// Program
+$publish_program = IGV_get_option('_igv_program_options', '_igv_publish_program');
+
+$program_image = IGV_get_option('_igv_program_options', '_igv_program_image');
+
+$program_text_en = IGV_get_option('_igv_program_options', '_igv_program_temp_text_en');
+$program_text_es = IGV_get_option('_igv_program_options', '_igv_program_temp_text_es');
+
+$start_date = IGV_get_option('_igv_site_options', '_igv_fair_start');
+$end_date = IGV_get_option('_igv_site_options', '_igv_fair_end');
+
+//Partners
+$partners = IGV_get_option('_igv_sponsors_options', '_igv_partners_group');
+
+?>
+
+<!-- main content -->
+
+<main id="main-content">
+
+<?php
+//
+// SPLASH
+//
+//
+
+if (!empty($splash_id)) { ?>
+  <section id="front-splash" class="section">
+    <?php 
+      echo (!empty($splash_link) ? '<a href="' . esc_url( $splash_link ) . '">' : ''); 
+      echo wp_get_attachment_image($splash_id, 'splash', null, array('class' => 'splash-image'));
```

Comment:
I thought this was going to be a post or something [or maybe a gallery/carousel of posts]? Just choosing image and link seems a bit weird. We could just have a post search input to choose anything